tephra-core/src/test/java/co/cask/tephra/TransactionServiceMainTest.java

```diff
@@ -58,6 +60,8 @@ public void run() {
 
       try {
         t.start();
+        // Wait for service to startup
+        TimeUnit.SECONDS.sleep(5);
```

Comment:
Why is the sleep necessary?  If there's a race, would be better to use an explicit signal or count down latch so we don't have to always wait 5 seconds.